classes/task/asynchronous_backup_task.php

```diff
@@ -130,7 +132,8 @@ private function before_backup_started_hook(\backup_controller $backup_controlle
                 'grade_histories' => false,
                 'users' => false,
                 'anonymize' => false,
-                'include_badges' => false
+                'include_badges' => false,
+                'filename' => 'backup-'. $item_entity->get_type() . '_' . $item_entity->get_timecreated() . '.mbz'
```

Comment:
I just realized we could also just use the sharing cart item id. It's completely unique and a simple method of ensuring files don't collide
```suggestion
                'filename' => 'sharing_cart_backup-'. $item_entity->get_id() . '.mbz'
```